script/generate_docs.sh

```diff
@@ -16,13 +16,13 @@ RSTS_DIR=`mktemp -d "${BASEDIR}/rsts_tmp/XXXXXXXXX"`
 
 # Copy all rst files to the same place
 cp -R rsts/* ${RSTS_DIR}
-cp -R _rsts/* ${RSTS_DIR}
+#cp -R _rsts/* ${RSTS_DIR}
```

Comment:
I don't remember, what does this do?

Reply:
i think we need this, good catch, added back